install.sh

```diff
@@ -423,6 +423,28 @@ function update_existing_chart() {
   echo 'Done! (note that images could still be downloading in the background...)'
 }
 
+function open_tensorleap_url() {
+  sleep 2 # Avoid server-is-down
+
+  TENSORLEAP_URL="http://127.0.0.1:4589"
+
+  if [ "$OS_NAME" == "Linux" ];
+  then
+    if type xdg-open > /dev/null; then
+      xdg-open $TENSORLEAP_URL
```

Comment:
After finally testing on linux, it seems there is a problem after all
While the browser open, the terminal gets different weird error messages

based on the first answer to this [question from stackoverflow](https://stackoverflow.com/questions/21084457/alter-behavior-of-xdg-open)
I think this should fix it
```suggestion
      xdg-open "$TENSORLEAP_URL"  2> /dev/null
```